### PR TITLE
fixed warnings after move to oxygen

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/editor/XbaseEditorOpenClassFileTest.java [hunk 1]
```diff
@@ -514,7 +514,7 @@ public IPackageFragmentRoot addJarToClassPath(final IJavaProject jp, final Strin
   public String getEditorContents(final IJavaElement javaElement) {
     try {
       final IEditorPart editor = JavaUI.openInEditor(javaElement);
-      final Object adapter = editor.getAdapter(IRewriteTarget.class);
+      final Object adapter = editor.<IRewriteTarget>getAdapter(IRewriteTarget.class);
```
Comment (szarnekow) [inline org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/editor/XbaseEditorOpenClassFileTest.java]: Should adapter be of type IRewriteTarget?

Comment (cdietrich) [reply org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/editor/XbaseEditorOpenClassFileTest.java]: It’s xtend gen so don’t know

Comment (szarnekow) [reply org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/editor/XbaseEditorOpenClassFileTest.java]: Should work if the local vars type is present in the Xtend file

**Comment by cdietrich** [comment]
@szarnekow @kthoms ok to merge?

**Pull request merged**